package.json

```diff
@@ -12,28 +12,27 @@
 		"url": "https://github.com/WordPress/wporg-mu-plugins/issues"
 	},
 	"devDependencies": {
-		"@wordpress/browserslist-config": "4.1.2",
-		"@wordpress/env": "5.5.0",
-		"@wordpress/scripts": "23.5.0",
+		"@wordpress/browserslist-config": "5.33.0",
+		"@wordpress/env": "9.2.0",
+		"@wordpress/scripts": "27.1.0",
 		"chalk": "4.1.2",
 		"cross-spawn": "7.0.3",
-		"cssnano": "5.1.11",
-		"node-watch": "^0.7.3",
-		"postcss": "8.4.14",
-		"postcss-cli": "9.1.0",
-		"postcss-import": "14.1.0",
-		"postcss-preset-env": "7.7.1",
+		"cssnano": "6.0.3",
+		"node-watch": "0.7.4",
+		"postcss": "8.4.33",
+		"postcss-cli": "11.0.0",
+		"postcss-import": "16.0.0",
+		"postcss-preset-env": "9.3.0",
 		"resolve-bin": "1.0.1",
-		"rtlcss": "3.5.0",
-		"url-loader": "^3.0.0",
-		"webpack": "5.47.1",
-		"webpack-cli": "4.9.1"
+		"rtlcss": "4.1.1",
+		"url-loader": "4.1.1"
```

Comment:
How did you arrive at this set of dependencies? It seems more extensive than expected.

Reply:
I used `yarn outdated` to update everything available. I wanted to make sure any peer dependencies were up to date with the new wordpress/ packages & supported the latest node version.